shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/type/DatabaseTypeFactoryTest.java

```diff
@@ -57,4 +57,17 @@ public void assertGetInstances() {
         assertThat(iterator.next(), instanceOf(SQLServerDatabaseType.class));
         assertThat(iterator.next(), instanceOf(H2DatabaseType.class));
     }
+    
+    @Test
+    public void assertGetDatabaseType() {
```

Comment:
Please rename the method to `assertGetInstance` if the target test method is `getInstance`